spec/models/video_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+
+describe Video do
+  it { should belong_to(:category)}
+  it { should validate_presence_of(:title)}
+  it { should validate_presence_of(:description)}
+
+  describe "search_by_title" do
```

Comment:
You should stick to RSpecs style on how to annotate `describe` blocks - check here http://betterspecs.org/#describe